modules/rgbd/src/odometry.cpp

```diff
@@ -1009,6 +1009,28 @@ Ptr<DepthCleaner> DepthCleaner::create(int depth_in, int window_size_in, int met
   return makePtr<DepthCleaner>(depth_in, window_size_in, method_in);
 }
 
+RgbdPlane::RgbdPlane(int method, int block_size,
```

Comment:
These definitions should be placed in plane.cpp

Reply:
Will do that in a new PR.